File: Individual_Assignment/code/python/matrix_multiplication.py
def multiply_matrixes( input1, input2, output,dim1,dim2,dim3):
    for i in range(dim1):
        for j in range(dim3):
            output[i][j] = 0

    for i in range(dim1):
        for k in range(dim2):
            for j in range(dim3):
                output[i][j] += input1[i][k] * input2[k][j]

File: Individual_Assignment/code/c/matrix_multiplication.c
void multiply_matrixes(const double *input1, const double *input2, double *output,
                     int dim1, int dim2, int dim3) {
    // Zeroing output matrix
    for(int i = 0; i<dim1;i++)
        for(int j = 0; j<dim3;j++)
            output[i*dim3 + j] = 0;
    // Calculating output matrix
    for (int i = 0; i < dim1; ++i)
        for (int k = 0; k < dim2; ++k)
            for (int j = 0; j < dim3; ++j)
                output[i * dim3 + j] += input1[i * dim2 + k] * input2[k * dim3 + j];
}
